client/src/pages/Profile/Availability/useStyles.tsx

```diff
@@ -0,0 +1,62 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  itemWrap: {
```

Comment:
These properties can be passed as props to the MUI Box component directly.

Eg: <Box display="flex" alignItems="center" maxWidth....>